sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/FunctionAppsTests.java

```diff
@@ -569,7 +573,7 @@ public void canCreateAndUpdateFunctionAppOnAcaWithPrivateRegistryImage() {
             .withManagedEnvironmentId(managedEnvironmentId)
             .withMaxReplicas(10)
             .withMinReplicas(3)
-            .withPrivateRegistryImage("xiaofeiacr.azurecr.io/samples/nginx:latest", "xiaofeiacr.azurecr.io")
+            .withPrivateRegistryImage("samples/nginx:latest", "https://xiaofeiacr.azurecr.io")
```

Comment:
ACA doesn't allow URL with `https` anymore... Previously we exposed that to the customer. Here we align the behavior with normal FunctionApp.